### ISSUE TITLE
Banners are used for gamecube games sometimes even though boxart was found and downloaded

### ISSUE THREAD

**Issue by Masamune3210** [GerbilSoft/rom-properties#270]
For some reason, even though the priority is set to Boxart first, sometimes gamecube games will use their internal banners instead of the boxart even if the boxart is downloaded
![image](https://user-images.githubusercontent.com/1053504/94465685-e8074200-0185-11eb-98fe-de9398017678.png)
![image](https://user-images.githubusercontent.com/1053504/94465711-efc6e680-0185-11eb-8f09-ba47d9ca205b.png)

**Comment by Amnesia1000** [comment]
Sometimes you can't find the cover, for example the ID "GF4S52" corresponds to the game "Los 4 fantasticos" in Spanish, but it is not in the gametdb database. What I do when that happens is change the ID from another version of the same game, for example GF4P52 and rename it to GF4S52. But it is not a problem of rom-properties, but the cover does not exist.

**Comment by Masamune3210** [comment]
In this issue, the box art is found, or at least I'm assuming it's found as it's shown in the Properties instead of the banner or no icon

**Comment by Amnesia1000** [comment]
Ahhh ok. Did you try from the rom properties configuration app to delete the windows cache and rom-properties?

**Comment by Masamune3210** [comment]
Yep, deleted windows cache both in rom properties and manually. For some reason the banner seems to be perferred for some reason

**Comment by GerbilSoft** [comment]
I haven't been able to reproduce this on KDE (Linux). It might be a Windows-specific issue; I'll test that tomorrow.

**Comment by Masamune3210** [comment]
I temp moved rom-properties cache to see if it was a issue with the cache, but no, same thing reappeared. If you havent figured it out by tomorrow ill try rebooting when this run is finished

**Comment by Amnesia1000** [comment]
I passed all my ISOs to rvz, and some covers do not show them. The data shown in the RVZ file is the same as that shown for the ISO. The thumbnails are downloaded. The system thumbnail cache and rom-property thumbnails were cleaned and are still not displayed.

Wii and GameCube alike.

Windows 10

![Captura de pantalla 2020-10-07 170308](https://user-images.githubusercontent.com/45115804/95382068-746cdf80-08bf-11eb-9a68-ee77cc7c5f2c.png)
